### PR TITLE
Add "Table of Contents" of README.md

### PR DESCRIPTION
**Pull request by PeterDaveHello**

### CODE REVIEW

**Comment by ryanoasis** [comment]
I like this :smile: . Anyone else?

File: README.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
...
+
+- [About](#about)
+- [Quick Start](#quick-start)
+- [Docs](#docs)
+- [Changelog](#changelog)
+- [People Using Vundle](#people-using-vundle)
+- [Contributors](#contributors)
+- [Inspiration & Ideas](#inspiration--ideas)
+- [Also](#also)
+- [TODO:](#todo)
```
Comment (ryanoasis) [inline README.md]: Can you remove the colon here? Probably could be removed from the heading as well

Comment (PeterDaveHello) [inline README.md]: done

**Comment by ryanoasis** [comment]
Thanks!

**Pull request merged**